paddle/fluid/framework/ir/conv_bn_fuse_pass.cc

```diff
@@ -92,13 +92,26 @@ void recompute_bias_and_weights(const Scope* scope,
   // Re-compute weight of conv2d from BN
   auto* weights = scope->FindVar(conv_weight->Name())->GetMutable<LoDTensor>();
   auto weights_shape = weights->dims();
-  auto weights_shape_2d = flatten_to_2d(weights_shape, 1);
+  auto weights_data = weights->mutable_data<float>(platform::CPUPlace());
+
+  // ConvTranspose weights are in IOHW format
```

Comment:
Wasn't here possible to do weights fusing using Eigen (similar like for regulr conv) ?

Reply:
I tried to figure out if it's possible but double data reorder will be needed in order to do that similarly to regular conv. It's because conv2d_transpose filter weights are in format IOHW what means that values that have to be multiplied by certain batch_norm calculated value aren't aligned together. To illustrate that let's assume our weight's vector is [1,1,1,1,1,1,1,1] and our batch norm values are [2,3]. For regular conv, fused weights will look: [2,2,2,2,3,3,3,3] and for conv_transpose: [2,2,3,3,2,2,3,3] (it's just a simple example). I'm also not an expert in Eigen so there may be better solution but I can't propose such.

Reply:
ok, looks reasonable to me